packages/frontend-2/lib/workspaces/composables/management.ts

```diff
@@ -417,6 +423,76 @@ export function useCreateWorkspace() {
   }
 }
 
+export const useWorkspaceSso = (params: { workspaceSlug: string }) => {
```

Comment:
Can we keep all of these in their own file? management.ts is already at ~400 LOC, SSO composables can have their own file